src/main/kotlin/com/avail/serialization/SerializerOperation.kt

```diff
@@ -902,8 +958,8 @@ enum class SerializerOperation constructor(
 	},
 
 	/**
-	 * A [tuple of integers][TupleDescriptor] whose values all fall in the range
-	 * 0..2^31-1.
+	 * A [tuple&#32;of&#32;integers][TupleDescriptor] whose values all fall in
```

Comment:
I think just the word "tuple" should be in the link, and "of integers" should be after the link.